master/src/ComputerZoomWidget.h

```diff
@@ -38,15 +38,26 @@ class ComputerZoomWidget : public QWidget
 	ComputerZoomWidget( const ComputerControlInterface::Pointer& computerControlInterface );
 	~ComputerZoomWidget() override;
 
+	VncViewWidget* vncView() const
```

Comment:
Where is this getter used?

Reply:
In the eventFilter to switch the viewport
```
m_vncView->setViewport( {} );
...
m_vncView->setViewport(screens[currentScreen].geometry);
```

Reply:
It accesses `m_vncView` directly (which is fine) and does not use the function ;-) So we can safely remove the function.

Reply:
Ah. I see. :)